OutOfSchool/OutOfSchool.IdentityServer/Data/Migrations/OutOfSchoolMigrations/20210617153920_ApplicationParentUpdate.cs

```diff
@@ -0,0 +1,76 @@
+﻿using Microsoft.EntityFrameworkCore.Migrations;
+
+namespace OutOfSchool.IdentityServer.Data.Migrations.OutOfSchoolMigrations
+{
+    public partial class ApplicationParentUpdate : Migration
```

Comment:
we have a new migration applied to develop so probably need to fix your to be on top of that migration :(

Reply:
changed migration to be the on the top